modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/LabelRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+interface IProps {
+	onClick: () => void;
+	value: LocalizedValue<string>;
+}
+
+export default function LabelRenderer({onClick, value}: IProps) {
+	return (
+		<div className="table-list-title">
+			<a href="#" onClick={onClick}>
+				{Object.keys(value).length !== 0 ? (
+					value
+				) : (
+					<ClayIcon symbol="view" />
+				)}
```

Comment:
Trying to replicate the behavior before the migration when using the `frontend-data-set:headless-display` taglib as you can see [here](https://github.com/carolmariaabb/liferay-portal/blob/9fec7027a7da0c35f3dcf118deb5691cf2a52330/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/ActionLinkRenderer.js#L156).